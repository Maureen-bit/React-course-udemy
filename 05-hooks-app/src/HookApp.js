import React from 'react';

export const HookApp = () => {
    return(
        <div>
            <h1> Hola Maureen </h1>
        </div>
    )
};