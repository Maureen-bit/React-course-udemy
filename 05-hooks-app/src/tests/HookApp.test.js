import { shallow } from 'enzyme';
import React from 'react';
import { HookApp } from '../HookApp';

describe('Test custom hook <HookApp />', () => {
    test('render <HookApp />', () => {
        const wrapper = shallow(<HookApp />);
        expect(wrapper).toMatchSnapshot();
    })
})