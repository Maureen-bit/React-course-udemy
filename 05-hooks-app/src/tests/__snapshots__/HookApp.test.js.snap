// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test custom hook <HookApp /> render <HookApp /> 1`] = `
<div>
  <h1>
     Hola Maureen 
  </h1>
</div>
`;
